### PR TITLE
update documentation

### PR DESCRIPTION
**Pull request by hassanfa**
This PR changes the documentation.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -101,3 +102,21 @@ Repository to collect and share bed files for each bait set. This repository doe
 
 - Hassan Foroughi
 - Anna Engström
+
+#### Versioning Scheme:
+
```
Comment (annaengstrom) [inline README.md]:
```suggestion
```

File: README.md [hunk 2]
```diff
@@ -101,3 +102,21 @@ Repository to collect and share bed files for each bait set. This repository doe
...
+- major: addition, updating, or removing following items from a bed file:
+	- Genome version
+	- Region size (even 1bp) (e.g. chr:Start-End)
+	- Target bed name (not file name). (e.g. GMCKSolid to GMCKsolid or GMCKsolidtumor)
+
+- minor: addition, updating, or removing following items:
+	- External URLs
+	- annotations and features
+	- file name (not panel name)
```
Comment (annaengstrom) [inline README.md]:
```suggestion
	- file name (i.e. not panel name)
```

Comment (annaengstrom) [inline README.md]: very nice!!!111!!!

File: README.md [hunk 3]
```diff
@@ -10,6 +10,7 @@ Repository to collect and share bed files for each bait set. This repository doe
 * [Panel contents](#panelcontent)
 * [Requirement to add new panel](#panelmetadata)
 * [Maintainers](#maintainers)
+* [Versioning Scheme](#versioning)
```
Comment (annaengstrom) [inline README.md]:
```suggestion
* [Versioning Schema](#versioning)
```

File: README.md [hunk 4]
```diff
@@ -101,3 +102,21 @@ Repository to collect and share bed files for each bait set. This repository doe
 
 - Hassan Foroughi
 - Anna Engström
+
+#### Versioning Scheme:
```
Comment (annaengstrom) [inline README.md]: Similar to above.

File: README.md (hunk 2 above)
Comment (hassanfa) [reply README.md]: Much appreciated!!!1111eleven!!!!

File: README.md (hunk 4 above)
Comment (hassanfa) [reply README.md]: Adressed you comments in latest commit!

**Pull request merged**